docs/backuprestore-with-oadp.md

```diff
@@ -0,0 +1,459 @@
+# Backup and restore with the Openshift API for Data Protection(OADP)
+
+## Overview
+
+The lifecycle Agent operator (LCA) provides functionality for backing up and restoring platform and application resources using OADP during image-based upgrades. During the upgrade stage, the LCA performs the following actions:
+
+Before the cluster is rebooted to the new stateroot:
+- Process the configmaps specified via `spec.oadpContent`.
+- Apply all backup CRs wrapped in the configmaps. If any backup CR fails, the upgrade process is terminated.
+- Export all restore CRs wrapped in the configmaps to the new stateroot.
+- Export the live DataProtectionApplication(DPA) CR and the associated secrets used in the DPA to the new stateroot.
+
+After the cluster is rebooted to the new stateroot:
+- Restore the preserved secrets and DPA
+- Apply all preverved restore CRs. If any restore CR fails, the upgrade process is terminated.
+
+## Pre-Requisites
+- A S3-compatible object storage must be set up and ensure that it's configured and accessible
+- OADP operator must be installed on both target and seed SNOs
+
+## LCA apply wave annotation
+
+The annotation `lca.openshift.io/apply-wave` is supported in the backup or restore CR to define the order in which the backup or restore CRs should be applied by LCA. The value of the annotation should be a string number, for example:
+```
+anntotations:
+  lca.openshift.io/apply-wave: "1"
+```
+If the annotation is provided in the backup or restore CRs, they will be applied in increasing order based on the annotation value. The LCA will move on the next group of CRs only after completing the previous group with the same wave number.
+If no `lca.openshift.io/apply-wave` annotation is defined in the backup or restore CRs, which means no particular order is required, they will be applied all together. 
+
+## Install OADP and configure OADP on target cluster via ZTP GitOps
+
+ Install OADP via [GitOps ZTP pipeline](https://docs.openshift.com/container-platform/4.14/scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.html).
```

Comment:
Should we have a section for non-gitops install? Or maybe a note here that the non-PGT bits can just be applied (minus the wave annotation), plus some site-specific customizations?